core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BootMonitors.scala

```diff
@@ -492,7 +500,7 @@ object BootMonitors extends LazyLogging {
             billingRepo,
             new BillingProjectDeletion(samDAO, billingRepo, billingProfileManagerDAO)
           )
-        )
+        ) ++ JobType.cloneJobTypes.map(jobType => jobType -> workspaceCloneRunner).toMap
```

Comment:
Map all clone job types to the clone runner, and concatenate it with existing mappings.